image-processing/cut_houses/cut_houses.py

```diff
@@ -224,9 +224,18 @@ def create_house_masks(image:str, output:str = None,)->np.ndarray:
 def cut_mask_from_image(image:str|np.ndarray, mask:str|np.ndarray, output:str = None, inverted:bool = False)-> np.ndarray:
 
     try:
+        # copy from core/preprocessing _load_image -> prevent import errors
```

Comment:
Shoulded than save and load moved to their own file and imported twice?

Reply:
hab ich jetzt so gemacht. Ich hoffe nur, dass niemand anderes jetzt import errors bekommt, weil ers von nem anderen pfad aus aufruft..